dotCMS/src/main/java/com/dotcms/graphql/business/NavigationFieldProvider.java

```diff
@@ -29,7 +29,7 @@ public enum NavigationFieldProvider implements GraphQLFieldsProvider {
     public Collection<GraphQLFieldDefinition> getFields() throws DotDataException {
         final GraphQLOutputType outputType = (GraphQLOutputType)NavigationTypeProvider.INSTANCE.getTypes().iterator().next();
         return Set.of(newFieldDefinition()
-                .name("Navigation")
+                .name("DotNavigationAPI")
```

Comment:
Types should represent data structures, not APIs. The "API" suffix suggests this is a service or endpoint rather than a data type. It's like naming a type "UserAPI" instead of "User".

Reply:
Agree.  Also should be a final constant, right?

I think `DotNavigation` would be enough.

Reply:
Done!